db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/BinaryRelationalOperatorNode.java

```diff
@@ -1513,13 +1517,19 @@ public double joinSelectivity(Optimizable optTable,
 
         if (rightOperand instanceof ColumnReference && ((ColumnReference) rightOperand).getSource().getTableColumnDescriptor() != null) {
             ColumnReference right = (ColumnReference) rightOperand;
+            if (!right.useRealColumnStatistics()) {
+                noStatsColumns.add(right.getSource().getSchemaName() + "." + right.getSource().getFullName());
```

Comment:
since we're using it here multiple times, maybe add function 

```
public String getSchemaTableName() throws StandardException
{
   return getSchemaName() + "." + getFullName();
}
```

to ResultColumn.java

Reply:
Done.